cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/setDetails.isml

```diff
@@ -55,6 +52,10 @@
              </div>
         </div>
         <isinclude template="product/components/setItems" />
+    
+        <isif condition="${dw.system.Site.current.preferences.custom.CLDGalleryEnabled}">
+            <script src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL}" onload="window.cldProductGallery = cloudinary; window.renderCloudinarySetGalleryWidgets();"></script>
```

Comment:
Please remove the legacy calls to renderCloudinarySetGalleryWidget from https://github.com/asad-rafter/cloudinary_sfcc_site_cartridge/blob/984285740779e611bacfdcb5f6cc623a559ff505/cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js#L95